CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### Visual Environment
 
+- [Node Searcher preserves its zoom factor.][3327]
```

Comment:
This needs additional explanation. Thunk about changelog as something read by users. They should understand what happened after reading these notes. The link to the task is "additional technical explanation", but without seeing it, it should already be clear.

Reply:
Added additional explanation, please take a look.